{
  "id": "CVE-2022-37451",
  "published": "2022-08-06T18:15:08",
  "modified": "2025-07-29T10:38:53.515633",
  "details": "Exim before 4.96 has an invalid free in pam_converse in auths/call_pam.c because store_free is not used after store_malloc.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
    }
  ],
  "repo_url": "https://github.com/exim/exim",
  "cwes": [],
  "commits": [
    "c1b32ab6ef9300e2ecab6736139e3e50874cd3a6",
    "fef95dbbb2220e10f5a6bf7ac1b6ecdff0025bca"
  ]
}
